xcube/core/store/fs/impl/dataset.py

```diff
@@ -304,3 +306,58 @@ def write_data(self,
         if not is_local:
             fs.put_file(file_path, data_id)
         return data_id
+
+
+GEOTIFF_OPEN_DATA_PARAMS_SCHEMA = JsonObjectSchema(
+    properties=dict(
+        # TODO: add more from ds.to_tiff()
```

Comment:
Does this exist: `ds.to_tiff()` ?

Reply:
Add schema for `tile_size`.

Reply:
@forman I don't understand what to add here exactly